front/app/components/admin/participation/index.tsx

```diff
@@ -1,25 +1,64 @@
 import React from 'react';
 
-import { useParams } from 'react-router-dom';
+import { Box, colors } from '@citizenlab/cl2-component-library';
+import { useLocation, useParams } from 'react-router-dom';
 
 import usePhases from 'api/phases/usePhases';
 
-import ParticipationDatesRange from './ParticipationDateRange';
+import NavigationTabs, {
+  Tab,
+  TabsPageLayout,
+} from 'components/admin/NavigationTabs';
+
+import { useIntl } from 'utils/cl-intl';
+import { defaultAdminCardPadding } from 'utils/styleConstants';
+
+import Demographics from './Demographics';
+import messages from './messages';
+import Users from './Users';
 
 const ProjectParticipation = () => {
   const { projectId } = useParams() as { projectId: string };
+  const { pathname } = useLocation();
+  const { formatMessage } = useIntl();
   const { data: phases } = usePhases(projectId);
 
   const startOfFirstPhase = phases?.data[0]?.attributes.start_at;
   const endOfLastPhase =
     phases?.data[phases.data.length - 1]?.attributes.end_at;
 
   if (!phases) return null;
+
+  const isDemographicsTab = pathname.includes('/demographics');
+
   return (
-    <ParticipationDatesRange
-      defaultStartDate={startOfFirstPhase}
-      defaultEndDate={endOfLastPhase ?? undefined}
-    />
+    <>
+      <NavigationTabs position="relative">
+        <Tab
+          label={formatMessage(messages.usersTab)}
+          url={`/admin/projects/${projectId}/participation`}
+          active={!isDemographicsTab}
```

Comment:
Are we expecting to add more tabs here soon?

Just if so - might be nice to have a more general "active" attribute we can use instead of relying on if it is/isn't the demographcis tab. Ignore this comment if not relevant though :)